gulpfile.js

```diff
@@ -24,6 +24,15 @@ const config = {
 	// Path to the default configuration file for bundler.
 	BUNDLE_DEFAULT_CONFIG: 'dev/bundles/build-config-standard.js',
 
+	DOCUMENTATION: {
+		// Path to the built editors for samples.
+		EDITORS_DIST: '.docs/assets/scripts/samples',
```

Comment:
During this issue, those editors aren't needed. But, in the future we will introduce `docs-builder` and that tool will need to have the editors (for sample pages).